src/index.js

```diff
@@ -1 +1,48 @@
 import './styles/styles.css';
+import 'regenerator-runtime/runtime';
+
+const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/QWe6fd15ksAlBbepEHeq/scores/';
+
+const refresh = document.querySelector('#refresh');
+const ul = document.querySelector('.ul');
+
+let arrayOfScores = [];
+
+async function getscores() {
+  ul.innerHTML = '';
+  const response = await fetch(baseUrl);
+  const data = await response.json();
+  arrayOfScores = data.result;
+
+  arrayOfScores.forEach((score) => {
+    const li = document.createElement('li');
+    li.innerHTML = `${score.user}: ${score.score}`;
+    li.className = 'list-item';
+    ul.append(li);
+  });
+}
+
```

Comment:
- Improve your ``form``  without alphanumeric. Try to implement numeric characters for your score.